src/devices/DCMotor/DCMotor.cs

```diff
@@ -116,20 +116,34 @@ public static DCMotor Create(int speedControlPin, GpioController controller = nu
         }
 
         /// <summary>
-        /// Creates <see cref="DCMotor"/> instance which allows to control speed in both directions.
+        /// Starts the motor by starting pwm with last speed set.
+        /// </summary>
+        public virtual void Start()
```

Comment:
isn't this equivalent to `Speed = 0` and `Speed != 0`? For this functional API change I'd recommend creating a separate issue

Reply:
I will create a new issue and explain what I wanted to achieve.